config/betterquesting/DefaultQuests/Quests/HandleLogisticswiththePipes-9990412c-63c3-4101-8355-f713ff087d01/TheBackboneoftheSystem-220faf6f-1f7e-4766-a50d-50326cf1b32c.json

```diff
@@ -0,0 +1,74 @@
+{
+  "questIDHigh:4": 2454373213744809830,
+  "preRequisites:9": {
+    "0:10": {
+      "questIDHigh:4": 0,
+      "questIDLow:4": 97
+    }
+  },
+  "questIDLow:4": -6553493705247837396,
+  "properties:10": {
+    "betterquesting:10": {
+      "snd_complete:8": "random.levelup",
+      "taskLogic:8": "AND",
+      "visibility:8": "NORMAL",
+      "isMain:1": 0,
+      "simultaneous:1": 0,
+      "icon:10": {
+        "id:8": "LogisticsPipes:item.PipeItemsBasicTransport",
+        "Count:3": 1,
+        "Damage:2": 0,
+        "OreDict:8": ""
+      },
+      "snd_update:8": "random.levelup",
+      "repeatTime:3": -1,
+      "globalShare:1": 0,
+      "questLogic:8": "AND",
+      "repeat_relative:1": 1,
+      "name:8": "The Backbone of the System",
+      "lockedProgress:1": 0,
+      "autoClaim:1": 0,
+      "isSilent:1": 0,
+      "desc:8": "The Unrouted Transport Pipe is the first pipe you will ever need in logistics pipes.\n\nIt only connects to other pipes and it doesn\u0027t have any item routing capabilities, but you will need a ton of it.\n\n§cWarning: Do NOT form junctions with the unrouted transport pipes. items going into an unrouted transport pipe junction will come out of a random exit of the junction."
+    }
+  },
+  "tasks:9": {
+    "0:10": {
+      "partialMatch:1": 1,
+      "autoConsume:1": 0,
+      "groupDetect:1": 0,
+      "ignoreNBT:1": 1,
+      "index:3": 0,
+      "consume:1": 0,
+      "requiredItems:9": {
+        "0:10": {
+          "id:8": "LogisticsPipes:item.PipeItemsBasicTransport",
+          "Count:3": 64,
+          "Damage:2": 0,
+          "OreDict:8": ""
+        }
+      },
+      "taskID:8": "bq_standard:retrieval"
+    }
+  },
+  "rewards:9": {
+    "0:10": {
+      "rewardID:8": "bq_standard:item",
+      "index:3": 0,
+      "rewards:9": {
+        "0:10": {
+          "id:8": "LogisticsPipes:item.PipeItemsBasicTransport",
+          "Count:3": 32,
```

Comment:
Too many for a reward i think, 16 or 8 sounds more reasonable in my opinion.

Reply:
how can I update this pull request?

Reply:
you edit https://github.com/905Sunny/GT-New-Horizons-Modpack with your changes, and it will edit this pull request